### PR TITLE
Change Docker Username

### PR DESCRIPTION
**Pull request by joaofigueiredobisect**

### CODE REVIEW

**Review by peterbrightwell — APPROVED** [review]
Looks right

File: README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ This will run the docker containers:
 
 ## Access the Development Container
```
Comment (massis08) [inline README.md:43]: Add to the readme an explanation of how it is possible to map a .conan2 folder in order to have persistent data storage.

**Review by massis08 — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -41,6 +41,8 @@ This will run the docker containers:
...
 
+This project includes a development container to facilitate code development. When the container is built for the first time, it is empty, and the repository needs to be downloaded again inside the container to build the project.
```
Comment (pedro-alves-ferreira) [inline README.md:44]: How about "cloned again" rather than "downloaded again"?

File: README.md [hunk 3]
```diff
@@ -53,7 +55,11 @@ First you need to check you IP address. You can do it by running:
 
 Once you know your ip address you enter the container by doing:
 
-    ssh -p 55555 bisect@{your-ip-address} -XY
+    ssh -p 55555 nmos@{your-ip-address} -XY
+
+The password for the username is:
```
Comment (pedro-alves-ferreira) [inline README.md:60]: The password for this user is

File: README.md [hunk 4]
```diff
@@ -81,6 +87,10 @@ If you have not used Conan before:
 
   conan profile detect --force
 
+- You can map an outside of the container .conan2 folder in order to have persistent data storage. You just need to add the bellow line on to the volumes inside the docker-compose-x86-development.yml.
```
Comment (pedro-alves-ferreira) [inline README.md:90]: "below", not "bellow"

**Review by pedro-alves-ferreira — CHANGES_REQUESTED** [review]

**Review by pedro-alves-ferreira — APPROVED** [review]
LGTM

**Pull request merged**